### PR TITLE
Add SSL support to Mongo connection

### PR DESCRIPTION
**Pull request by mattdodge**

### CODE REVIEW

**Comment by mattdodge** [comment]
This bumps the base block to version `3.0.0` because it is a breaking change (configuration for connection is wrapped up in object now) and also because the `Find` block was on its own version (`2.0.0`). So this removes that independent versioning and bumps them all with a major version bump.

**Comment by mattdodge** [comment]
This fixes #34

File: mongodb_base.py [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+    SelectProperty, VersionProperty, BoolProperty
 
 __all__ = ['MongoDBBase', 'Limitable', 'Sortable']
 
 
+class Connection(PropertyHolder):
+
+    """ User credentials
+    Properties:
+        username (str): User to connect as
```
Comment (hansmosh) [inline mongodb_base.py]: copy pasta

Comment (mattdodge) [inline mongodb_base.py]: gah, you caught me!!

**Pull request merged**